server/models/index.js

```diff
@@ -0,0 +1,54 @@
+//both: name, email, password, rating, city
+//customer: jobs: name, description, required skills, 
+//need date, city, job by id, proposals by id
+//contractor: skills, jobs completed.  Proposals: contractor
+//name, cost estimate, start date estimate, time frame estimate
+
```

Comment:
To use mongoose.Schema and mongoose.model need to import mongoose package. Applicable to all model files. 
```suggestion
const {Schema, model} = require('mongoose');
```
can destructure to just use Schema and mode directly